website/docs/desktop-testing/Electron.md

```diff
@@ -28,34 +28,73 @@ The configuration wizard will install all required packages and creates a `wdio.
 
 That's it 🎉
 
+## Configuration
+
+To use the service you need to add `electron` to your services array and set an Electron capability, e.g.:
+
+```js
+// wdio.conf.js
+import url from 'node:url';
+import path from 'node:path';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+export const config = {
+  outputDir: 'logs',
+  // ...
+  services: ['electron'],
+  capabilities: [{
+      browserName: 'electron'
+  }],
+  // ...
+};
+```
+
+The service will attempt to find the path to your bundled Electron application if you use [Electron Forge](https://www.electronforge.io/) or [Electron Builder](https://www.electron.build/) as bundler. You can provide a custom path to the binary via custom service capabilities, e.g.:
+
+```ts
+capabilities: [{
+  browserName: 'electron',
+  'wdio:electronServiceOptions': {
+    appBinaryPath: './path/to/bundled/electron/app.exe',
+    appArgs: ['foo', 'bar=baz'],
+  },
+}],
+```
+
 ## API Configuration
 
 If you wish to use the electron APIs then you will need to import (or require) the preload and main scripts in your app. Somewhere near the top of your preload:
 
 ```ts
+const isTest = process.env.NODE_ENV === 'test'
 if (isTest) {
-  import('wdio-electron-service/preload');
+  require('wdio-electron-service/preload');
 }
 ```
 
 And somewhere near the top of your main index file (app entry point):
 
 ```ts
+const isTest = process.env.NODE_ENV === 'test'
 if (isTest) {
-  import('wdio-electron-service/main');
+  require('wdio-electron-service/main');
```

Comment:
I noticed this change in the boilerplate project too, are there issues using dynamic import here?

Reply:
This I believe fails if your Electron project is forced to use a CJS setup in your `tsconfig.json`. AFAIK the Electron ecosystem still has no good support for ESM. While it is not impossible to get it working (looking at the service example), I wanted to make sure to not have it failing either due to type issues and using require here is just a safe way to demonstrate it.